include/ofi.h

```diff
@@ -326,6 +326,16 @@ static inline void *ofi_get_page_end(const void *addr, size_t page_size)
 			+ page_size, page_size) - 1);
 }
 
+static inline bool ofi_is_size_aligned(size_t size, size_t alignment)
+{
+	return ((size % alignment) == 0);
+}
+
+static inline bool ofi_is_addr_aligned(void *addr, size_t alignment)
+{
+	return (addr == ofi_get_page_start(addr, alignment));
+}
```

Comment:
Because here you're using the word 'aligned' to mean something else.  The implementation is confusing me as to the intent.  ofi_get_page_start() takes a page_size as its second parameter.  'Alignment' suggests a byte alignment value, not a page size.  Couldn't this just be:
```
(((uintptr_t) addr) % alignment) == 0)
```

Reply:
Thx, updated.